Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -4778,6 +4778,10 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
     case CSSPropertySizeAdjust:
         return nullptr;
 
+    // Unimplemented @view-transition properties.
```

Comment:
```suggestion
    // @view-transition descriptors.
```

nit: Probably more accurate to say they're not supported because they're descriptors.